rails-i18n.gemspec

```diff
@@ -15,8 +15,8 @@ Gem::Specification.new do |s|
   s.require_path = 'lib'
   s.required_rubygems_version = '>= 1.8.11'
 
-  s.add_dependency('i18n', '>= 0.7', '< 2')
-  s.add_dependency('railties', '>= 6.0.0', '< 8')
+  s.add_runtime_dependency('i18n', '>= 0.7', '< 2')
+  s.add_runtime_dependency('railties', '>= 6.0.0', '< 9')
```

Comment:
Should there be an upper constraint at all?

Reply:
I don't think so; was trying to introduce the least amount of friction to get this approved. 

If maintainers are okay with it I'd love to remove the upper constraint!

Reply:
Just remove it. Upper constraints are almost always a bad idea.

Reply:
This upper dependency require to do the same dance every few years:
  - https://github.com/svenfuchs/rails-i18n/commit/c797b4e3dae1a735ec57fb5d7bda49aa4a5bc2be
  - https://github.com/svenfuchs/rails-i18n/commit/306461b19cb367d44eba2fd4b210bda51a63347e
  - https://github.com/svenfuchs/rails-i18n/commit/85a8b273de40b425b75a933500e330c11f2f6e35

That prevent people from trying Rails release candidates etc. It's just a bad pattern.

Reply:
Done

Reply:
Exactly. Unfortunately some maintainers still believe it's safer to restrain the upper limit. which leads me to maintain a lot of forks every year

Reply:
Thank you for removing this!!!